modules/c1xBidAdapter.md

```diff
@@ -0,0 +1,64 @@
+# Overview
+
+Module Name: C1X Bidder Adapter
+Module Type: Bidder Adapter
+Maintainer: vishnu@c1exchange.com
+
+# Description
+
+Module that connects to C1X's demand sources
+
```

Comment:
Please include a table describing your available params, their values and associating types, along with a description. `floorPriceMap` should be included.

Reply:
added floor price in the test param fields cc @vishnu1211

Reply:
Thank you! Your recent commit also adds,`bidfloor` as a possible param.